templates/mentorship_relation_accepted.html

```diff
@@ -0,0 +1,11 @@
+<p>Hi {{ sender }},</p>
+<br>
+<p>Congratulations!!</p>
+<br>
+<p>Your mentorship request has been accepted by {{ receiver }}.
```

Comment:
Can we link to the receiver profile on the receiver name?

Reply:
I'm not sure about linking, but I've added some more data from the request to enhance the email.